src/advancedKubectl/content.md

```diff
@@ -0,0 +1,1108 @@
+.example-2:hover ~ .abbreviations .alias-sys,
+.example-2:hover ~ .abbreviations .alias-rm,
+.example-2:hover ~ .abbreviations .alias-cm,
+
+.example-3:hover ~ .abbreviations .alias-k,
+.example-3:hover ~ .abbreviations .alias-sys,
+.example-3:hover ~ .abbreviations .alias-g,
+.example-3:hover ~ .abbreviations .alias-svc,
+.example-3:hover ~ .abbreviations .alias-w,
+
+.example-4:hover ~ .abbreviations .alias-k,
+.example-4:hover ~ .abbreviations .alias-g,
+.example-4:hover ~ .abbreviations .alias-dep,
+.example-4:hover ~ .abbreviations .alias-sl,
+.example-4:hover ~ .abbreviations .alias-all
+{
+  color: #E7040F;
+}
+</style>
+```
+
+As you can see, the aliases consist of **components**, each standing for a specific element of a kubectl command. Each alias can have one component for the **base command**, **operation**, and **resource**, and multiple components for the **options**, and you just "fill in" these components from left to right according to the above scheme.
+
+> Please note that the current and fully detailed scheme is on the [GitHub page](https://github.com/ahmetb/kubectl-aliases#syntax-explanation). There you can also find the [full list of aliases](https://github.com/ahmetb/kubectl-aliases/blob/master/.kubectl_aliases).
+
+For example, the alias `kgpooyamlall` stands for the command `kubectl get pods -o yaml --all-namespaces`:
+
+```include
+<template>
+<ul class="list pl0 pr5 flex justify-center f-headline b pointer mv4">
+<li class="hover-sky relative hide-child pv2">k<span class="child absolute bottom--2 left-0 f2">kubectl</span></li>
+<li class="hover-sky relative hide-child pv2">g<span class="child absolute bottom--2 left-0 f2">get</span></li>
+<li class="hover-sky relative hide-child pv2">po<span class="child absolute bottom--2 left-0 f2">pods</span></li>
+<li class="hover-sky relative hide-child pv2">oyaml<span class="child absolute bottom--2 left-0 f2" no-wrap>-o yaml</span></li>
+<li class="hover-sky relative hide-child pv2">all<span class="child absolute bottom--2 left-0 f2 no-wrap">--all-namespaces</span></li>
+</ul>
+</template>
+<script>
+</script>
+```
+
+Note that the relative order of most option components doesn't matter. So, `kgpooyamlall` is equivalent to `kgpoalloyaml`.
+
+You don't need to use all the components for an alias. For example, `k`, `kg`, `klo`, `ksys`, or `kgpo` are valid aliases too. Furthermore, you can combine aliases with other words on the command-line.
+
+For example, you could use `k proxy` for running `kubectl proxy`:
+
+```include
+<template>
+<ul class="list pl0 pr5 flex justify-center f-headline b pointer mv4">
+<li class="hover-sky relative hide-child pv2">k <span class="child absolute bottom--2 left-0 f2">kubectl</span></li>
+<li class="relative hide-child pv2 gray">&nbsp;proxy</li>
+</ul>
+</template>
+<script>
+</script>
+```
+
+Or you could use `kg roles` for running `kubectl get roles` (there exists currently no alias component for the Roles resource):
```

Comment:
_(it doesn't exist an alias for the Roles component at the moment)_

Reply:
Changed wording.